server.js

```diff
@@ -4,24 +4,58 @@ require('dotenv').config();
 const express = require('express');
 const cors = require('cors');
 const mongoose = require('mongoose');
-// const Book = require('./model/book');
+const { handle } = require('express/lib/application');
```

Comment:
Good work on workin through the lab!

- Do we need this line? This looks like it was autogenerated...